pkg/controller/servicebindingrequest/envvars/envvars_test.go

```diff
@@ -111,6 +111,31 @@ func TestBuild(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "should create envvar for int64 type",
+			expected: map[string]string{
+				"STATUS_VALUE": "-9223372036",
+			},
+			src: map[string]interface{}{
+				"status": map[string]interface{}{
+					"value": int64(-9223372036),
+				},
+			},
+		},
+		{
+			name: "should create envvar for float64 type",
+			expected: map[string]string{
+				"": "100.72",
+			},
+			src: float64(100.72),
+		},
+		{
+			name: "should create envvar for empty string type",
```

Comment:
@isutton Shouldn't we avoid this scenario?
The code should take care of an empty string passed to `src`.

Reply:
Shouldn't this return an error instead?